lib/archethic/beacon_chain.ex

```diff
@@ -414,4 +416,55 @@ defmodule Archethic.BeaconChain do
         e
     end
   end
+
+  # Slots which are already has been added
+  # Real time transaction can be get from pubsub
+  def list_transactions_from_current_slots(date = %DateTime{} \\ DateTime.utc_now()) do
```

Comment:
If you can add typespecs this would be great !

Reply:
yes